packages/preview/illc-mol-thesis/0.1.0/LICENSE

```diff
@@ -0,0 +1,170 @@
+Creative Commons Attribution-ShareAlike 4.0 International
```

Comment:
We generally don't allow CC license for code. It is only allowed for images and other assets. Could you please re-license your project under a license that is more appropriate for software?

Reply:
Thanks. I re-licensed in d1abee7. I initially misunderstood the README and thought my package would not qualify as "real" code.

Reply:
It does!